src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/ViewModels/Teams/TeamViewModel.cs

```diff
@@ -88,12 +90,28 @@ public static TeamViewModel Map(Team team)
         /// <returns> Domain object </returns>
         public Team ToDomain()
         {
-            return new Team {
-                Id = Id,
-                Name = Name,
-                Coach = Coach,
-                Achievements = Achievements,
-                Captain = CaptainId
+            return new Team(Id,
+                Name,
+                Coach,
+                Achievements,
+                new PlayerId(CaptainId),
+                Roster == null ? new List<PlayerId>() :
```

Comment:
Can you make sure that Roster is not null always? So you can avoid this check

Reply:
change auto property for default init empty list